maven/bnd-export-maven-plugin/src/main/java/aQute/bnd/build/StandaloneRun.java

```diff
@@ -0,0 +1,25 @@
+package aQute.bnd.build;
```

Comment:
I remember we discuss that this was in a bnd package to access something. Lets change that something and move this class into the aQute.bnd.maven.export.plugin package. What needs to change in bndlib to do this?